superbench/benchmarks/base.py

```diff
@@ -8,6 +8,8 @@
 from datetime import datetime
 from abc import ABC, abstractmethod
 
+import numpy as np
```

Comment:
add it in dependency? it's only implicitly depended by torch

Reply:
Added